src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml

```diff
@@ -133,6 +134,7 @@ entity:
         prefered_language: Preferred language attribute
         personal_code: Personal code attribute
         scoped_affiliation: Scoped affiliation attribute
+        organization_unit: Organization unit
```

Comment:
Given we no longer support `oidc` entitites, this entry can be omitted. And the entire section can be wiped I assume. But feel free to only refrain from adding the organizationion_unit here.